runtime/hydradx/src/xcm.rs

```diff
@@ -248,7 +251,7 @@ impl orml_unknown_tokens::Config for Runtime {
 
 impl orml_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type SovereignOrigin = MoreThanHalfCouncil;
+	type SovereignOrigin = EitherOf<EnsureRoot<Self::AccountId>, GeneralAdmin>;
```

Comment:
i would consider restricting this to the root track only, as it controls balance of the parachain